apps/account-api/test/e2e-setup.mock.spec.ts

```diff
@@ -21,18 +21,18 @@ import { u8aToHex } from '@polkadot/util';
 export const FREQUENCY_API_WS_URL = process.env.FREQUENCY_API_WS_URL || 'ws://0.0.0.0:9944';
 export const BASE_SEED_PHRASE = process.env.SEED_PHRASE || '//Alice';
 
-export async function setupProviderAndUsers(numUsers = 4) {
+export async function setupProviderAndUsers(seedPhrase = BASE_SEED_PHRASE, numUsers = 4) {
```

Comment:
Maybe better to make `seedPhrase` the second argument so existing calls don't need to change? (And maybe `baseSeedPhrase` is a better argument name)

Reply:
These mods aren't technically needed at all if we change this all to be a unit test (see my other comment), but it does still seem like a worthy enhancement to this setup function.